src/crimson/osd/pg.cc

```diff
@@ -936,8 +951,32 @@ PG::submit_transaction(
     projected_log.add(entry);
   }
 
+  std::vector<pg_shard_t> shards_to_send_txn;
+  std::vector<pg_shard_t> shards_to_send_logs;
+  std::vector<pg_shard_t> to_push_clone;
+  const auto &pg_shards = peering_state.get_acting_recovery_backfill();
+  for (auto &pg_shard : pg_shards) {
+    if (pg_shard == pg_whoami) {
+      continue;
+    }
+    auto &hoid = obc->obs.oi.soid;
+    if (should_send_op(pg_shard, hoid)) {
+      shards_to_send_txn.push_back(pg_shard);
+    } else {
+      shards_to_send_logs.push_back(pg_shard);
+      if (_new_clone && is_missing_on_peer(pg_shard, hoid)) {
+        // The head is in the push queue but hasn't been pushed,
+        // we need to also add the newly created clone to the push queue;
+        // otherwise, the clone has no chance of being pushed although the
+        // head does.
```

Comment:
nit
```
        // The head is in the push queue but hasn't been pushed yet.
        // We need to ensure that the newly created clone will be 
        // pushed as well, otherwise we might skip it.
        // See: https://tracker.ceph.com/issues/68808
```